tests/conftest.py

```diff
@@ -24,32 +23,60 @@
 def TestModel():
     """Provides a test model"""
 
-    class TestModel(MarshModel):
-        class Table:
-            name = 'peanut-butter'
-            hash_key = 'foo'
-            range_key = 'bar'
-            read = 5
-            write = 5
+    if 'marshmallow' in (os.getenv('SERIALIZATION_PKG') or ''):
```

Comment:
Am I reading this correct in that the sole reason this giant conditional block exists is because for schematics we need `child = types.Map(types.String)` and for marshmallow we need `child = types.Map()`?

I think we need to find a way to shim the functionality differences so that the TestModel definition remains the same.